pkg/registry/contractRegistry_test.go

```diff
@@ -1,4 +1,4 @@
-package registry
+package registry_test
```

Comment:
Needed to remove circular dependencies - otherwise this test (in the registry package) depends on the mocks package, which depends on the registry package